lib/timodule.js

```diff
@@ -287,6 +296,15 @@ function find(modulesOrParams, platforms, deployType, sdkVersion, searchPaths, l
 								return;
 							}
 
+							if (moduleAPIVersion && moduleAPIVersion[platform] && info.manifest && info.manifest.apiversion && ~~info.manifest.apiversion < ~~moduleAPIVersion[platform]) {
```

Comment:
I don't think we can do a less than comparison. For example, I highly doubt that Android supports v1, v2, and v3 modules.